setup.py

```diff
@@ -1,108 +0,0 @@
-# SPDX-License-Identifier: BSD-3-Clause
-# Copyright (c) 2023 Scipp contributors (https://github.com/scipp)
-# @author Simon Heybrock
-from setuptools import find_packages
-from skbuild import setup
-
-
-def get_version():
-    import subprocess
-
-    return (
-        subprocess.run(
-            ['git', 'describe', '--tags', '--abbrev=0'], stdout=subprocess.PIPE
```

Comment:
Remind me how this gets the version now? Was it that code snipped we have in our `__init__.py` files?

Reply:
The same as all other packages. It uses `setuptools_scm` which essentially runs the code here. Differently from the other projects, here, we write the version to `src/scipp/_version.py` to allow scikit-build to pick it up. (I don't know how this works in pure setuptools-based builds.) However, the actual `scipp.__version__` is based on C++. There is some code in CMakeLists.txt - that I didn't touch - that also runs this git command and makes it so the version gets written into the binary. So the change here is only for the version reported in PyPI.

Reply:
> It uses setuptools_scm which essentially runs the code here.

Which code here? The file was removed.

Reply:
This code you highlighted to query git.

Reply:
So by "runs the code here", you do not mean the *code here* (which was deleted) but *runs the same command*?

Reply:
Yes, or a similar one. It uses git to get the latest tag and extract a version from it. (We've been using it in our python-only projects. But without writing a `_version.py` file.)